### DISCUSSION TITLE
Intersection environement: reward assigned twice

### DISCUSSION THREAD

**Discussion by francissunny25** [eleurent/highway-env#356] [Q&A] [answered]
https://github.com/eleurent/highway-env/blob/742ce2e69bf00169f2b250e0388c6b9926c026ec/highway_env/envs/intersection_env.py#L82

https://github.com/eleurent/highway-env/blob/742ce2e69bf00169f2b250e0388c6b9926c026ec/highway_env/envs/intersection_env.py#L83

Does this mean reward is always normalised regardless of the normalise flag in default config?

**Comment by eleurent** [comment, marked as answer]
I would not say that the reward is _always normalised_: it is set to `"arrived_reward"` when the goal is reached, and to the sum of other rewards terms otherwise. 
But this `"arrived_reward"` could be any value that you like, e.g. 10.0, while enabling `"normalize_reward"` remaps the reward range to the default [0,1]
